src/module-elasticsuite-indices/Model/Healthcheck/GhostIndicesCheck.php

```diff
@@ -53,7 +54,7 @@ public function __construct(
         UrlInterface $urlBuilder,
         int $sortOrder = 10
     ) {
-        parent::__construct($urlBuilder, $sortOrder);
+        parent::__construct($urlBuilder, $sortOrder, MessageInterface::SEVERITY_MINOR);
```

Comment:
Please add the severity is as an exposed constructor parameter, after the sortOrder.